src/Breadcrumb.php

```diff
@@ -13,6 +13,11 @@
  */
 final class Breadcrumb implements \JsonSerializable
 {
+    /**
+     * This constant defines the default breadcrumb type.
+     */
+    public const TYPE_DEFAULT = 'default';
```

Comment:
Apart from the rest of the PR, this addition should be retained. Thanks for spotting this.